### PR TITLE
add Paragraph borders

### PR DESCRIPTION
**Pull request by Griklit**
I added paragraph borders, however, there is some trouble:

1. due to the life cycle of XML-rs, it's hard to add attr dynamically to the extent that it's hard to add an Option type to the border, and I can't seem to solve this problem at my current level. I've left some commented out code in there, as well as the parameter space which should logically be of type Option<usize> as well.

2. something is inexplicably wrong with my linux environment, so much so that this code was written on windows, I'm assured that the paragraph borders work fine, but I can't test & update the snapshot.

### CODE REVIEW

File: docx-core/src/documents/elements/paragraph_property.rs [hunk 1]
```diff
@@ -37,6 +38,7 @@ pub struct ParagraphProperty {
     pub(crate) div_id: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub paragraph_property_change: Option<ParagraphPropertyChange>,
+    pub borders: Option<ParagraphBorders>,
```
Comment (bokuweb) [inline docx-core/src/documents/elements/paragraph_property.rs:41]: Could you please add `#[serde(skip_serializing_if = "Option::is_none")]`

```suggestion
    #[serde(skip_serializing_if = "Option::is_none")]
    pub borders: Option<ParagraphBorders>,
```

**Comment by bokuweb** [comment]
@Griklit Thanks for your great work!!

**Comment by Griklit** [comment]
> @Griklit Thanks for your great work!!

So, is there any way to add xml attributes dynamically?

**Comment by bokuweb** [comment]
@Griklit 

> So, is there any way to add xml attributes dynamically?

Currently there is no way

**Review by bokuweb — APPROVED** [review]
LGTM

**Pull request merged**